subprojects/docs/src/docs/userguide/compatibility.adoc

```diff
@@ -24,10 +24,14 @@ Java 6 and 7 can still be used for <<building_java_projects.adoc#sec:java_cross_
 Any supported version of Java can be used for compile or test.
 
 == Kotlin
-Gradle is tested with Kotlin 1.3.21 through 1.4.30.
+Gradle is tested with Kotlin 1.3.72 through 1.4.30.
 
 == Groovy
 Gradle is tested with Groovy 1.5.8 through 2.5.12.
 
 == Android
+<<<<<<< HEAD
 Gradle is tested with Android Gradle Plugin 4.0, 4.1, 4.2 and 7.0. Alpha and beta versions may or may not work.
+=======
```

Comment:
I fixed it in https://github.com/gradle/gradle/commit/4d0bcf182fa91e61249b9d3934f4c8eb4b853461